hyperspy/tests/misc/test_compilers.c

```diff
@@ -0,0 +1,4 @@
+int main(void)
```

Comment:
I am not sure if `hyperspy/tests/misc/` is the best place to put this file as this is not part of an unittest. Beside, I think that c files are currently ignored when packaging. Should this individual file be declared in MANIFEST?